runtime/test-runtime/src/lib.rs

```diff
@@ -24,8 +24,9 @@ use pallet_transaction_payment::CurrencyAdapter;
 use sp_std::prelude::*;
 use codec::Encode;
 use polkadot_runtime_parachains::{
-	configuration,
+	configuration as parachains_configuration,
```

Comment:
oh, that's unfortunate, does this have anything to do with added `Config`?

Reply:
No, just wanted similar naming as in rococo.

Reply:
ah, ok, 
we should go over all of them then.

It's ok for the later PR though.